packages/test/test-service-load/src/utils.ts

```diff
@@ -104,18 +129,37 @@ export async function initialize(testDriver: ITestDriver, seed: number) {
                 },
             }),
         options,
+        detachedBlobStorage: new MockDetachedBlobStorage(),
     });
 
     const container = await loader.createDetachedContainer(codeDetails);
     container.on("error", (error) => {
         console.log(error);
         process.exit(-1);
     });
+
+    if ((testConfig.detachedBlobs ?? 0) > 0) {
+        const ds = await requestFluidObject<ILoadTest>(container, "/");
+        const dsm = await ds.detached({
+            runId: -1,
+            testConfig,
+            verbose: true,
+            randEng,
+        });
+
+        // await Promise.all(Array(testConfig.detachedBlobs).fill(0).map(async () => { await dsm.writeBlob(); }));
+        await Promise.all(Array(testConfig.detachedBlobs).fill(0).map(async () => dsm.writeBlob()));
+    }
+
     const testId = Date.now().toString();
     const request = testDriver.createCreateNewRequest(testId);
     await container.attach(request);
     container.close();
 
+    if (testDriver.type === "odsp") {
```

Comment:
why do we need special odsp handling here? the test driver should do the right thing

Reply:
When attachment blobs are present, detached containers are created with a .tmp extension, which causes the test drivers not to work correctly since the filename is not `${testId}.fluid`. (attaching a container with attachment blobs is not supported on other drivers)

Reply:
this functionality should be in the test driver iteself

Reply:
#7684